### DISCUSSION TITLE
completions correct work after 'exec zsh'

### DISCUSSION THREAD

**Discussion by finegorko** [ohmyzsh/ohmyzsh#11029] [Help] [answered]
for example, when i start terminal and type `brew install spot`

and press TAB - nothing, only dir like a `'Desktop/`, but when i `exec zsh` everything magically starts working

**Comment by rwmitchell** [comment]
before you type ```exec zsh``` what does ```echo $SHELL``` show ?

**Comment by finegorko** [comment, marked as answer]
nvm, this fixed my problem
```
if type brew &>/dev/null; then
  FPATH=$(brew --prefix)/share/zsh/site-functions:$FPATH

  autoload -Uz compinit
  compinit
fi
```
